src/scenes/Languages/Detail/LanguageDetail.graphql

```diff
@@ -75,3 +75,14 @@ mutation RemoveLocationFromLanguage($languageId: ID!, $locationId: ID!) {
     ...LanguageForm
   }
 }
+
+query LanguagePostListOverview($language: ID!, $input: PostListInput) {
+  language(id: $language) {
+    id
+    ... on Postable {
+      posts(input: $input) {
+        ...PostList
+      }
+    }
```

Comment:
```suggestion
    posts(input: $input) {
      ...PostList
    }
```
The specific call out on the interface isn't needed since `Language` implements `Postable`.

This was previously needed on the project query because `Project` interface didn't implement `Postable`.
I fixed the API yesterday to have that, so it's no longer needed on that query now either.